lib/controllers/dojo/save.spec.js

```diff
@@ -96,6 +97,82 @@ lab.experiment('save a dojo', function () {
       done();
     });
   });
+  // TODO : test not changing upon equality
+  lab.describe('the events should be notified', function () {
```

Comment:
Similarly to above, should probably more be like 'event addresses should be updated'